analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S4581.html

```diff
@@ -10,6 +10,8 @@ <h2>Noncompliant Code Example</h2>
 public void Foo()
 {
     var g = new Guid(); // Noncompliant - what's the intent?
+    var guid = default(Guid);
+    Guid other = default;
```

Comment:
Here also. Probably this one will get fixed automatically when the other one is fixed.